### ISSUE TITLE
Insufficient color contrast for email link on applicant page

### ISSUE THREAD

**Issue by jeanniefu** [civiform/civiform#3734]
Color contrast between for link text must be at least 3:1. The email link contrast is only 2.41:1.

Current UI:
![image](https://user-images.githubusercontent.com/9094240/198741766-9518a63d-b6d4-41fd-900c-a010697db4fc.png)

My proposal is to update the email link styling to match other links on the page. @mferhut - Does that sound good to you?

Proposed UI:
![image](https://user-images.githubusercontent.com/9094240/198742078-47d6a4e6-bf68-4df2-ab86-4542082fd4b9.png)

------
Details for eng:

This is failing on renovate bot's PR for upgrading axe-core testing framework version: https://github.com/civiform/civiform/pull/3713

Example failure message: 
```
    [
      {
        "id": "link-in-text-block",
        "impact": "serious",
        "tags": [
          "cat.color",
          "wcag2a",
          "wcag141"
        ],
        "description": "Ensure links are distinguished from surrounding text in a way that does not rely on color",
        "help": "Links must be distinguishable without relying on color",
        "helpUrl": "https://dequeuniversity.com/rules/axe/4.5/link-in-text-block?application=axeAPI",
        "nodes": [
          {
            "any": [
              {
                "id": "link-in-text-block",
                "data": {
                  "messageKey": "fgContrast",
                  "contrastRatio": 2.407591263211715,
                  "requiredContrastRatio": 3,
                  "nodeColor": {
                    "red": [30](https://github.com/civiform/civiform/actions/runs/3339298104/jobs/5528072260#step:9:31),
                    "green": 64,
                    "blue": 175,
                    "alpha": 1
                  },
                  "parentColor": {
                    "red": 0,
                    "green": 0,
                    "blue": 0,
                    "alpha": 1
                  }
                },
                "relatedNodes": [
                  {
                    "html": "<div class=\"mx-auto max-w-screen-sm w-5/6\">For technical support, contact <a href=\"mailto:CiviForm@seattle.gov\" target=\"_blank\" class=\"text-blue-800\">CiviForm@seattle.gov</a></div>",
                    "target": [
                      ".max-w-screen-sm"
                    ]
                  }
                ],
                "impact": "serious",
                "message": "The link has insufficient color contrast of 2.40759126[32](https://github.com/civiform/civiform/actions/runs/3339298104/jobs/5528072260#step:9:33)11715:1 with the surrounding text. (Minimum contrast is 3:1, link text: [object Object], surrounding text: [object Object])"
              },
              {
                "id": "link-in-text-block-style",
                "data": null,
                "relatedNodes": [
                  {
                    "html": "<div class=\"mx-auto max-w-screen-sm w-5/6\">For technical support, contact <a href=\"mailto:CiviForm@seattle.gov\" target=\"_blank\" class=\"text-blue-800\">CiviForm@seattle.gov</a></div>",
                    "target": [
                      ".max-w-screen-sm"
                    ]
                  }
                ],
                "impact": "serious",
                "message": "The link has no styling (such as underline) to distinguish it from the surrounding text"
              }
            ],
            "all": [],
            "none": [],
            "impact": "serious",
            "html": "<a href=\"mailto:CiviForm@seattle.gov\" target=\"_blank\" class=\"text-blue-800\">CiviForm@seattle.gov</a>",
            "target": [
              ".text-blue-800"
            ],
            "failureSummary": "Fix any of the following:\n  The link has insufficient color contrast of 2.[40](https://github.com/civiform/civiform/actions/runs/3339298104/jobs/5528072260#step:9:41)7[59](https://github.com/civiform/civiform/actions/runs/3339298104/jobs/5528072260#step:9:60)12[63](https://github.com/civiform/civiform/actions/runs/3339298104/jobs/5528072260#step:9:64)211[71](https://github.com/civiform/civiform/actions/runs/3339298104/jobs/5528072260#step:9:72)5:1 with the surrounding text. (Minimum contrast is 3:1, link text: [object Object], surrounding text: [object Object])\n  The link has no styling (such as underline) to distinguish it from the surrounding text"
          }
        ]
      }
    ]
```

**Comment by mferhut** [comment]
Hi @jeanniefu yes that sounds good. All links should be one a11y blue color. 

Do we support dark theme on mobile/desktop based on the device/browser settings?

**Comment by jeanniefu** [comment]
Thank you! We do not support dark theme.
